### DISCUSSION TITLE
Is there a way of using non-english languages?

### DISCUSSION THREAD

**Discussion by Pol-Lanski** [pietrop/digital-paper-edit-electron#70] [language]
This tool is amazing. I've tested it and it would be ***just what I need*** for my Spanish-speaking podcast. Transcript, add notes/paper-edit, send to editor and voilà!

Unfortunately right now it seems to only support English. Is there any workaround? Anyone in the community has managed to use another model for Mozilla DeepSpeech or similar?

That would **really** help me :)

Regardless of whether I can use it for this spanish project, good work @pietrop !

**Comment by pietrop** [comment]
Thansk @Pol-Lanski 

unfortunately with speechmatics discontinuing the web API portal https://github.com/pietrop/digital-paper-edit-electron/issues/57 we lost support for multiple languages. In autoEdit2 we used to have IBM but we have been having issues porting this to autoEdit3 https://github.com/pietrop/digital-paper-edit-electron/issues/51 one possibility could be to add sypport for Google STT https://github.com/pietrop/digital-paper-edit-electron/issues/49

These are some related queries re adding Spanish https://github.com/pietrop/digital-paper-edit-electron/issues/62 https://github.com/pietrop/digital-paper-edit-electron/issues/60

unfortunately at the moment I don't have a lot of spare time to look into this, focused more on consolidating other parts of the application. If you do have capacity and know how for a PR, I am happy to provide more info and advise tho. Let me know.
